### ISSUE TITLE
Full screen channels drop down list

### ISSUE THREAD

**Issue by tomidis** [Me-TV/Me-TV#19]
In full screen mode, the channels drop down list doesn't work always.
It's hard to describe this issue, but often happens disappearing of control bar with channels list (after you click on some channel). It seems like there's a problem with keeping the control bar on the screen. Then if you wait a few second, the control bar (and selecting the channels) again works.

**Comment by russel** [comment]
Activating the drop down menu seems not to inhibit the time out of the control bar.

**Comment by tomidis** [comment]
Yes, I think so

**Comment by russel** [comment]
I doubt I can attack this problem for the next couple of days, nor the mouse problem, but I'll get stuck into it over the weekend.

**Comment by tomidis** [comment]
just take your time

**Comment by tomidis** [comment]
I can confirm, the same happens also with sound volume button.
Furthermore, I was clicking on the sound volume button, and suddenly the control bar disappeared there wasn't it anymore. Then I find out that Me-TV was freezed, the TV program continued, but Me-TV was not reacting on any command, even the "adaptor" window was not possible to close.
When I closed it using task bar, I get the message "There was an error reported on the GStreamer bus. Internal data stream error."

**Comment by russel** [comment]
I understand the problem with the volume button, it is directly related to the control bar timeout problem – which is a problem that has been around since the first C++ move from GTK+2 to GTK+3, I never fixed in in the C++, D, or now the Rust version, I must now do this!

I suspect the GStreamer data stream error is an indirect problem. At this stage I think it is ignorable. Only once the control bar timeout problem is fixed, so that the volume widget works correctly, would it be worrying if there was still a GStreamer stream error.

**Comment by russel** [comment]
Should be fixed by changeset https://github.com/Me-TV/Me-TV/commit/e7206fe164066aa376aac36cf2218f5c8487ed6e if this is not the case, best to open a new issue rather than continue with this one.
